pages/linux/scriptreplay.md

```diff
@@ -0,0 +1,15 @@
+# scriptreplay
+
+> Output content of session files created by script command to standard output.
+
+- Replay {{path/to/session.out}} at the speed it was recorded:
+
+`scriptreplay {{path/to/timingfile}} {{path/to/session.out}}`
+
+- Replay {{typescript_file}} at double-speed:
```

Comment:
```suggestion
- Replay a typescript at double the original speed:
```
I'd emphasize that's double *of the original* speed, same goes for the example below with half the original speed.

Reply:
Well said!